docs/source/masked.rst

```diff
@@ -0,0 +1,10 @@
+torch.masked
+============
+
+.. automodule:: torch.masked
+.. automodule:: torch.masked.maskedtensor
+
+Introduction
```

Comment:
Note that the doc page looks like this: https://docs-preview.pytorch.org/82836/masked.html

Should this point to the masked repo? Or we're happy to leave it as is until the next PR up comes in?

Reply:
Ah good idea - I'll add a diff after the core functionality to get it updated!